public/index.html

```diff
@@ -94,12 +107,16 @@ <h2>PayJoin here to open these channels</h2>
 				document.querySelector(':root').style.setProperty('--primary', "rgb(" + shifted + ")");
 				document.querySelector(':root').style.setProperty('--secondary', "rgb(" + color_triplet + ")");
 			}
-
-			function add_channel() {
-				let channels = document.getElementById("channels");
+			
+			function add_channel(address=""){
+				const channels = document.getElementById("channels");
 				let indexToAdd = Math.floor(channels.children.length/2)
 				let div = document.createElement('div');
-				div.innerHTML = `<input type="text" name="channels[${indexToAdd}][node]" aria-labelledby="destinationLabel" placeholder="node@host:port" span="1" required><input type="number" name="channels[${indexToAdd}][amount]" min="20000" step="1" aria-labelledby="capacityLabel" span="1" required>`;
+				if (address == "") {
+					div.innerHTML = `<input type="text" name="channels[${indexToAdd}][node]" aria-labelledby="destinationLabel" placeholder="node@host:port" span="1" required><input type="number" name="channels[${indexToAdd}][amount]" min="20000" step="1" aria-labelledby="capacityLabel" span="1" required>`;
+				} else {
+					div.innerHTML = `<input type="text" value="${address}" name="channels[${indexToAdd}][node]" aria-labelledby="destinationLabel" placeholder="node@host:port" span="1" required><input type="number" name="channels[${indexToAdd}][amount]" min="20000" step="1" aria-labelledby="capacityLabel" span="1" required>`;
+				}
 				channels.appendChild(div.firstChild);
 				channels.appendChild(div.lastChild);
```

Comment:
perhaps focus div.lastChild while you have the handle

Reply:
Sure